website/source/guides/security/acl.html.markdown

```diff
@@ -356,6 +357,46 @@ region. When replication is interrupted, the existing data is used for request p
 become stale. When the authoritative region is reachable, replication will resume and repair any
 inconsistency.
 
+### host_volume Rules
+
+The `host_volume` policy controls access to mounting and accessing host volumes.
```

Comment:
Maybe we should cross-link "host volumes" to the docs for configuring host volumes? It seems like that's been done for the other rule sets above.

Reply:
I'm not sure where the best place to link to would be yet - I think I'd like to punt that until we add a bit of "summary" documentation.